app-commons/src/main/scala/org/bitcoins/commons/rpc/Command.scala

```diff
@@ -20,20 +39,1537 @@ trait AppServerCliCommand extends CliCommand {
   override def defaultPort: Int = 9999
 }
 
+trait Broadcastable extends CliCommand {
+  override def defaultPort: Int = 9999
+}
+trait SignDLCCliCommand extends AppServerCliCommand{
+  override def defaultPort: Int = 9999
+}
+
+trait AddDLCSigsCliCommand extends AppServerCliCommand{
+  override def defaultPort: Int = 9999
+}
+
+trait AcceptDLCCliCommand extends AppServerCliCommand{
+  override def defaultPort: Int = 9999
+}
+
+trait SendCliCommand extends AppServerCliCommand{
+  override def defaultPort: Int = 9999
+}
```

Comment:
These should all get `defaultPort` from `AppServerCliCommand` so you dont need to override here